apps/opik-backend/src/main/resources/liquibase/db-app-state/migrations/000004_add_prompt_library_tables.sql

```diff
@@ -0,0 +1,33 @@
+--liquibase formatted sql
+--changeset thiagohora:add_prompt_library_tables
+
+CREATE TABLE prompts (
+    id CHAR(36) NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    description VARCHAR(255),
+    created_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6),
+    created_by VARCHAR(100) NOT NULL DEFAULT 'admin',
+    last_updated_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6),
+    last_updated_by VARCHAR(100) NOT NULL DEFAULT 'admin',
+    workspace_id VARCHAR(150) NOT NULL,
+    CONSTRAINT `prompts_pk` PRIMARY KEY (id),
+    CONSTRAINT `prompts_workspace_id_name_uk` UNIQUE (workspace_id, name)
+);
+
+CREATE TABLE prompt_versions (
+    id CHAR(36) NOT NULL,
+    prompt_id CHAR(36) NOT NULL,
+    commit VARCHAR(7) NOT NULL,
```

Comment:
Are we going to use newly generated values for this field, or will it just be filled with part of the ID field value? 
Because if it’s the latter, it could be virtual column with reversed? copy of ID (it could be also like 7-8 chars)

Reply:
What is a good idea? I just wonder about the likelihood of collisions when using such a method.